benches/tinystr.rs

```diff
@@ -78,6 +78,40 @@ fn construct_from_str(c: &mut Criterion) {
     bench_block!(c, "construct_from_str", cfs);
 }
 
+fn construct_from_u8(c: &mut Criterion) {
+    macro_rules! cfu {
+        ($r:ty) => {
+            |b, inputs: &Vec<&str>| {
+                let raw: Vec<&[u8]> = inputs.iter().map(|s| s.as_bytes()).collect();
```

Comment:
I wouldn't do this and just do `for u in inputs.iter() { ... from_utf8(u.as_bytes() }` or such.